### ISSUE TITLE
event emitted when message is actually sent

### ISSUE THREAD

**Issue by siddo420** [primus/primus#408]
In the docs, it says:
```
When you are sending messages to the server, you don't have to wait for the open event to happen, the client will automatically buffer all the data you've send and automatically write it to the server once it's connected. The client supports a couple of different events.
```
The questions are then: Does primus emit an event when it sends a buffered message to the server at a later time? And/or, is it possible to prevent messages from being buffered by primus so they could be resent later by the application itself?

This is important if the app wants to track which messages were actually sent and exactly when.

**Comment by 3rd-Eden** [comment]
You could probably use the internal m ssages for this. The outgoing::data etc

Arnout Kazemier

**Comment by siddo420** [comment]
thanks ... will try and let u know

however, I assume there is no way to disable auto-buffering of messages

**Comment by 3rd-Eden** [comment]
You can disable the queue size and it should stop buffering. You can also check the readyState as OPEN is always sending unbuffered

Arnout Kazemier

**Comment by siddo420** [comment]
Right, but that'll make the change global, whereas I needed to disable buffering for a special class of messages only.

Thanks.
